cfme/tests/middleware/deployment_methods.py

```diff
@@ -18,8 +18,8 @@
 
 DEPLOYMENT_URL = 'http://{}:8180/{}'
 WAR_EXT = '.war'
-DELAY = 10
-NUM_SECS = 120
+DELAY = 30
+NUM_SECS = 600
```

Comment:
**Question**
No issue increasing overall timeout, but would like to hear some justification for the magnitude of the increase.  

Is there a potential bug or RFE here, to improve MIQ/CFME performance?  What used to work in two minutes now may take 10?

Reply:
the answer is in above comment ^^